### DISCUSSION TITLE
Auto login Functionality

### DISCUSSION THREAD

**Discussion by IIvexII** [IIvexII/LGU-TimetableAPI#19] [Q&A] [answered]
Why don't I add functionality to login automatically?

**Comment by IIvexII** [comment, marked as answer]
It was in my consideration but there is a reCaptcha to solve. I have tried many things to bypass this but failed. If you can bypass it, please do it and open a PR, I will test and merge it.

**Reply by Zain-ul-din** [reply]
I'm also looking for same solution
